solr_conf/conf/solrconfig.xml

```diff
@@ -749,6 +750,7 @@
       <str name="keyword_pf">
         description_t^50
         shelfmark_t^100
+        shelfmark_textnum^140
```

Comment:
Is there ever a science to our boosting? Would it be easier to sort these by their boosting levels?

Reply:
maybe? no, it's not very scientific — open to suggestions if you have thoughts on how to improve it